basic-algorithms-&-datastructures/index060.js

```diff
@@ -0,0 +1,37 @@
+const newsFeed = {
+    page: "front_page",
+      articles: [{
+        headline: "Programming: Expectations vs. Reality",
+        image: [],
+        date_published: "2019-01-05T08:00:00+08:00",
+        date_modified: "2019-01-05T09:20:00+08:00",
+        author: {
+          username: "PeoplePerson",
+          name: "Pat Comb"
+        },
+        likes: 10571,
+        dislikes: 14,
+        shares: 54,
+        comments: 461,
+        section: {
+          name: "Programming",
+          subscribers: 18101
+        },
+        url: "http://www.binary-bugle.com/programming-expectations-vs-reality"
+      },{},{},{},{},{}
+    ]}
+
+/*
+
+Now that we are getting the hang of the structure code of our database, we have autocompleted the rest of database.
```

Comment:
I'm not sure if `index060.js` and `index070.js` are the best places to review dot and bracket notation. It might be better to include a quick review in a later step when they're actually used to access or set a value, but that's just my quick take.